src/cf-forms.less

```diff
@@ -56,15 +60,32 @@
         <label class="form-label-header">
             Form label header
         </label>
+    - name: Form label helper text
+      markup: |
+        <label class="form-label-helper-text">
+            Form label helper text
+        </label>
   tags:
     - cf-forms
 */
 
 .form-label-header {
-    .h5();
+    .h4();
```

Comment:
This change should be happening in cf-core's `label` styling. `.form-label-header` is a separate pattern for when you need a heading above the form label, e.g. "Date range" would be the `.form-label-header` above two fields labeled "From" and "To".